cmd/root.go

```diff
@@ -589,3 +589,19 @@ func login(ctx context.Context, cmd *cobra.Command, scopes []string) (context.Co
 
 	return ctx, oi, token, nil
 }
+
+func getAppUrl(frontend, app string) (string, error) {
+	if frontend == "" && app == "" {
+		return "", fmt.Errorf("'--app' flag must be set")
```

Comment:
Does it really? Defaulting to `"https://app.overmind.tech"` is the right thing to do for all users (except those who know to provide something anyways).